src/transaction/creation_common.rs

```diff
@@ -895,6 +896,78 @@ mod delegation_rate {
     }
 }
 
+
+mod minimum_delegation_amount {
+    use super::*;
+    use casper_client::cli::CliError;
+    pub const ARG_NAME: &str = "minimum-delegation-amount";
+    const ALIAS: &str = "min-amount";
+    const ARG_VALUE_NAME: &str = common::ARG_INTEGER;
+    const ARG_HELP: &str = "the minimum delegation amount for the add-bid transaction";
+
+    pub fn arg() -> Arg {
+        Arg::new(ARG_NAME)
+            .long(ARG_NAME)
+            .value_name(ARG_VALUE_NAME)
+            .alias(ALIAS)
+            .help(ARG_HELP)
+            .required(true)
+            .display_order(DisplayOrder::MinimumDelegationRate as usize)
+    }
+
+    pub fn get(matches: &ArgMatches) -> &str {
+        matches
+            .get_one::<String>(ARG_NAME)
+            .map(String::as_str)
+            .unwrap_or_default()
+    }
+
+    pub(super) fn parse_delegation_amount(value: &str) -> Result<u64, CliError> {
+        value
+            .parse::<u64>()
+            .map_err(|err| CliError::FailedToParseInt {
+                context: "Add Bid: Minimum delegation amount",
+                error: err,
+            })
+    }
+}
+
+
+mod maximum_delegation_amount {
+    use super::*;
+    use casper_client::cli::CliError;
+    pub const ARG_NAME: &str = "maximum-delegation-amount";
+    const ALIAS: &str = "max-amount";
+    const ARG_VALUE_NAME: &str = common::ARG_INTEGER;
+    const ARG_HELP: &str = "the maximum delegation amount for the add-bid transaction";
+
+    pub fn arg() -> Arg {
+        Arg::new(ARG_NAME)
+            .long(ARG_NAME)
+            .value_name(ARG_VALUE_NAME)
+            .alias(ALIAS)
+            .help(ARG_HELP)
+            .required(true)
+            .display_order(DisplayOrder::MaximumDelegationRate as usize)
+    }
+
+    pub fn get(matches: &ArgMatches) -> &str {
+        matches
+            .get_one::<String>(ARG_NAME)
+            .map(String::as_str)
+            .unwrap_or_default()
+    }
+
+    pub(super) fn parse_delegation_amount(value: &str) -> Result<u64, CliError> {
+        value
+            .parse::<u64>()
+            .map_err(|err| CliError::FailedToParseInt {
+                context: "Add Bid: Maximum delegation amount",
```

Comment:
```suggestion
                context: "Add Bid: delegation amount",
```

Because I think this may happen for both "minimum" and "maximum".